parsers/imageAltTag.js

```diff
@@ -0,0 +1,39 @@
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('parser:imageAltTag');
+var parse = require('./lib/parse');
+
+/*
+ * Extract alt text from img tags
+ */
+
+var PARSER_COMPLETED_SUCCESSFULLY = false;
+
+function imageAltTag(snippet) {
+
+    debug("→ %j", _.omit(snippet, ['html']));
+
+    var $ = cheerio.load(snippet.html);
+
+    var altTagFounds = $('img[alt]');
+    altTagFounds = _.map(altTagFounds, function (e) {
+      return e.attribs.alt;
+    });
+    altTagFounds = _.flatten(altTagFounds);
+    altTagFounds = _.uniq(altTagFounds);
+
+    PARSER_COMPLETED_SUCCESSFULLY = true;
+    return {
+        imageAltTag: PARSER_COMPLETED_SUCCESSFULLY,
+        altTexts: altTagFounds
+    }
+};
+
+return parse.please({
+    'name': 'imageAltTag',
+    'requirements': { hrefType: 'photo' },
+    'implementation': imageAltTag,
+    'since': "2017-03-15",
```

Comment:
This is actually used whenever the parser will broke, in the future.

if feedAltImage has to change, we are not going to update this code, but rather adding and "end" data so the parser will not fail again.

and a new one get created, marked with `since` the data in which the facebook html code is changed enough to justify a new parser.